### PR TITLE
[KOA-4943] Upgrade Backpack to Storybook 6

### PR DESCRIPTION
**Pull request by anambl**
- Update Backpack to Storybook v6:
  - Upgrade `webpack` to version `^4.46.0`
  - Update storybook configuration following the guidelines [here](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#to-mainjs-configuration):
    - Delete `.storybook/config.js`
    - Rename `.storybook/addons.js` to `.storybook/main.js`
    - Add `.storybook/preview.js` to inject global stylesheets, fonts etc, into the preview bundle
    - Enable TypeScript stories
  - Update webpack configuration:
    - Add loader for React components which are using svg icons
    - Remove IE 11 config
  - Update story format as old format is no longer supported
  - Update all examples to follow the `{example-name}Example` format

Not in this PR:
- Enable storyshots
- Fix `Autoprefixer` warnings

PR to update docs site will follow.

### CODE REVIEW

File: .storybook/webpack.config.js
Comment (olliecurtis) [inline .storybook/webpack.config.js:40]: 😍

**Review by olliecurtis — COMMENTED** [review]
LGTM! Just one minor thing I would like to try before hitting the old merge button to see as it might also give us perf gains!

**Review by olliecurtis — APPROVED** [review]

File: .storybook/webpack.config.js
Comment (anambl) [reply .storybook/webpack.config.js:48]: Tried this and it's not working 😞

**Pull request merged**
